### DISCUSSION TITLE
how do I customize tailwind.config.js for apps/web?

### DISCUSSION THREAD

**Discussion by Learning-X** [vercel/turborepo#3747] [Help] [answered]
I'm currently using turborepo with-tailwind template and having trouble with tailwind.config.js while customizing the default theme for my turbo project. I'm currently doing this under packages/tailwind-config/tailwind.config.js, is this the right way of doing this in order to make this work in apps/web and so on...?
Thanks in advance.

**Comment by Learning-X** [comment]
> I'm currently using turborepo with-tailwind template and having trouble with tailwind.config.js while customizing the default theme for my turbo project. I'm currently doing this under packages/tailwind-config/tailwind.config.js, is this the right way of doing this in order to make this work in apps/web and so on...? Thanks in advance.

ANYONE? Would greatly appreciated.

**Comment by anthonyshew** [comment, marked as answer]
Yes, that sounds like it should work.
